versioned/mongodb/src/main/java/com/dremio/nessie/versioned/store/mongodb/MongoDBAliasCollectorImpl.java

```diff
@@ -43,16 +46,16 @@ public String escape(String unescaped) {
   public String alias(Entity value) {
     switch (value.getType()) {
       case MAP:
-        throw new UnsupportedOperationException("Unable to convert: " + value.getType());
+        return "{ $map {" + value.getMap().entrySet().stream().map(e -> String.format("%s: %s", e.getKey(), e.getValue()))
```

Comment:
Can you have a map of arbitrary values? What if it's a map of lists, that doesn't seem like it would be handled here...

Reply:
I pretty much took this from the Dynamo code AttributeValueUtil.fromEntity which is called from the corresponding implementation as this method. Can you accept this on the basis that is behaves the same as the current implementation?

Reply:
I updated this for MAP to call `alias` again with the value. If it is a LIST it will expand that out.